braille-printer-app/brf-printer.h

```diff
@@ -0,0 +1,498 @@
+#include <pappl/pappl.h>
+#include <cupsfilters/filter.h>
+#include <ppd/ppd-filter.h>
+
+pappl_content_t brf_GetFileContentType(pappl_job_t *job);
+
+extern int brf_JobIsCanceled(void *data);
```

Comment:
This is only used inside `brf-printer-app.c`, so you can make it `static` and not declare it here.